app/components/metrics_summary_component/view.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module MetricsSummaryComponent
+  class View < ViewComponent::Base
+    attr_accessor :start_date, :end_date, :weekly_submissions, :weekly_starts, :weekly_started_but_not_completed, :weekly_completion_rate, :form_has_metrics, :error_message
+
+    def initialize(metrics_data)
+      super
+      set_dates
+
+      if metrics_data.nil?
+        @error_message = I18n.t("metrics_summary.errors.error_loading_data_html")
+      elsif metrics_data[:form_is_new]
+        @error_message = I18n.t("metrics_summary.errors.new_form_html")
+      else
+        @weekly_submissions = metrics_data[:weekly_submissions]
+      end
+    end
+
+    def render?
+      FeatureService.enabled?(:metrics_for_form_creators_enabled)
+    end
+
+    def formatted_date_range
+      start_date_format_string = start_date.year == end_date.year ? "%e %B" : "%e %B %Y"
+      I18n.t("metrics_summary.date_range", start_date: start_date.strftime(start_date_format_string).strip, end_date: end_date.strftime("%e %B %Y").strip)
+    end
+
+  private
+
+    def set_dates
+      today = Time.zone.today
+      @start_date = today - 8.days
+      @end_date = today - 1.day
```

Comment:
```suggestion
      @start_date = 1.week.ago
      @end_date = 1.day.ago
```

Reply:
Had to tweak this slightly but good spot! Also made me realise that I was originally off by one on the start date